### PR TITLE
remove intercom

### PR DESCRIPTION
**Pull request by Vadman97**
## Summary

Replace intercom with discord callouts.
This change allows us to centralize the support to discord while maintaining
code to request help in the app.

## How did you test this change?

Reflame preview.

## Are there any deployment considerations?

No

## Does this work require review from our design team?

No

### CODE REVIEW

File: frontend/src/util/window.ts [hunk 1]
```diff
@@ -1,42 +1,16 @@
...
 	return FRONTEND_URI
 }
 
-interface IntercomInit {
-	admin?: Admin
+export const showSupportBubble = () => {
+	window.open('https://highlight.io/community', '_blank')
 }
 
-export function loadIntercom({ admin }: IntercomInit = {}) {
```
Comment (SpennyNDaJets) [inline frontend/src/util/window.ts:15]: Looks like we are still trying to import this function from a couple files
- frontend/src/index.tsx
- frontend/src/routers/AppRouter/AppRouter.tsx

File: frontend/src/util/window.ts [hunk 2]
```diff
@@ -1,42 +1,16 @@
...
-		})
-}
-
-export const showIntercomBubble = () => {
-	window.Intercom('update', { hide_default_launcher: false })
-}
-
-export const showIntercomMessage = (message = '') => {
-	window.Intercom('showNewMessage', message)
+export const showSupportMessage = async (_ = '') => {
```
Comment (SpennyNDaJets) [inline frontend/src/util/window.ts:13]: Looks like we aren't using the message that is provided in the method call (e.g. `'Hi! I would like to use the session export feature.'`), and wanted to make sure that this was intended

**Review by SpennyNDaJets — CHANGES_REQUESTED** [review]
Requesting changes for the export errors, but also added a question around the message

File: frontend/src/util/window.ts (hunk 1 above)
Comment (Vadman97) [reply frontend/src/util/window.ts:15]: whoops, good catch!

File: frontend/src/util/window.ts (hunk 2 above)
Comment (Vadman97) [reply frontend/src/util/window.ts]: yeah, wasn't sure what to best do with it; going to add it to the popup

**Comment by Vadman97** [comment]
Screenshot 2023-11-20 at 3 53 23 PM
Screenshot 2023-11-20 at 3 53 34 PM

**Review by SpennyNDaJets — APPROVED** [review]
LGTM!

**Pull request merged**